other/check-if-using-csi-secrets-store-driver-cve-2023-2878/check-if-using-csi-secrets-store-driver-cve-2023-2878.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: check-if-using-csi-secrets-store-driver
+  annotations:
+    policies.kyverno.io/title: Prevent CVE-2023-2878
+    policies.kyverno.io/category: Security
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod, Secret, CSI-Driver
+    kyverno.io/kyverno-version: 1.11.0
+spec:
+  validationFailureAction: Audit
+  rules:
+  - name: check-if-using-csi-secrets-store-driver
+    match:
+      resources:
+        kinds:
+        - csidriver
```

Comment:
Pretty sure this kind isn't correct. They are case sensitive.

Reply:
`k get csidrivers.storage.k8s.io` i am missing the `s` though

Reply:
A `kubectl get` operates differently from matching kinds. Singular vs. plural also matters. Have you actually tested this yourself with this resource?

Reply:
i have tested in the playground, where it does pass?

Reply:
You need to test this in a real cluster. Playground doesn't behave the same way when it comes to kind matching.